docs/source/configuration/parameters.md

```diff
@@ -0,0 +1,146 @@
+
+```python
+from kedro.config import ConfigLoader
+from kedro.framework.project import settings
+
+conf_path = str(project_path / settings.CONF_SOURCE)
+conf_loader = ConfigLoader(conf_source=conf_path, env="local")
+parameters = conf_loader["parameters"]
+```
+
+This loads configuration files from any subdirectories in `conf` that have a filename starting with `parameters`, or are located inside a folder with name starting with `parameters`.
+
+```{note}
+Since `env` is set to `local`, the configuration path `conf/local` takes precedence in the example above. Any overlapping top-level keys from `conf/base` are overwritten by those from `conf/local`.
+```
+
+Calling `conf_loader[key]` in the example above will throw a `MissingConfigException` error if no configuration files match the given key. But if this is a valid workflow for your application, you can handle it as follows:
+
+```python
+from kedro.config import ConfigLoader, MissingConfigException
+from kedro.framework.project import settings
+
+conf_path = str(project_path / settings.CONF_SOURCE)
+conf_loader = ConfigLoader(conf_source=conf_path, env="local")
+
+try:
+    parameters = conf_loader["parameters"]
+except MissingConfigException:
+    parameters = {}
+```
+
+```{note}
+The `kedro.framework.context.KedroContext` class uses the approach above to load project parameters.
+```
+
+[Parameters can then be used on their own or fed in as function inputs](#use-parameters).
+
+## How to specify parameters at runtime
+
+Kedro also allows you to specify runtime parameters for the `kedro run` CLI command. Use the `--params` command line option and specify a comma-separated list of key-value pairs that will be added to [KedroContext](/kedro.framework.context.KedroContext) parameters and made available to pipeline nodes.
+
+Each key-value pair is split on the first colon or equals sign. The following examples are both valid commands:
+
+```bash
+kedro run --params=param_key1:value1,param_key2:2.0  # this will add {"param_key1": "value1", "param_key2": 2} to parameters dictionary
+```
+```bash
+kedro run --params=param_key1=value1,param_key2=2.0
+```
+Values provided in the CLI take precedence and overwrite parameters specified in configuration files.
+
+* Parameter keys are _always_ treated as strings.
+* Parameter values are converted to a float or an integer number if the corresponding conversion succeeds; otherwise, they are also treated as string.
+
+If any extra parameter key and/or value contains spaces, wrap the whole option contents in quotes:
+
+```bash
+kedro run --params="key1=value with spaces,key2=value"
+```
+
+Since key-value pairs are split on the first colon or equals sign, values can contain colons/equals signs, but keys cannot. These are valid CLI commands:
+
+```bash
+kedro run --params=endpoint_url:https://endpoint.example.com
+```
+```bash
+kedro run --params=endpoint_url=https://endpoint.example.com
+```
+
+## How to use parameters
```

Comment:
I started to read this page wondering "what are parameters, or what are there _for_?", and then this last how-to solved it for me. Maybe we could move it to the very beginning, or use part of this text as an explanation?

Reply:
I've added an intro about parameters.